src/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface ChromaticConfig {
+  chromatic: ChromaticParameters;
+}
+
+export interface ChromaticParameters {
+  diffIncludeAntiAliasing?: boolean;
+  diffThreshold?: number;
+  disableE2EAutoCapture?: boolean;
+  pauseAnimationAtEnd?: boolean;
+}
+
+export interface ChromaticStorybookParameters extends ChromaticParameters {
+  viewports: number[];
+}
```

Comment:
What's the purpose of putting this in a separate interface? It goes in the actual chromatic object same as the other values.

Reply:
That's because `ChromaticParameters` is the interface for setting Chromatic params from the Playwright config, and we can't support the `viewports` option from there. But we do generate and pass `viewports` to the Storybook parameters, along with the other Chromatic params from the Playwright config. Is there a better way to handle that?